src/containers/Tenant/Query/QueryResult/QueryResultViewer.tsx

```diff
@@ -180,6 +182,21 @@ export function QueryResultViewer({
         );
     };
 
+    const renderQueryInfoDropdown = () => {
+        return (
+            <QueryInfoDropdown
+                queryResultsInfo={{
+                    ast: data.ast,
+                    stats: data.stats,
+                    queryText,
```

Comment:
lets' move `queryText` on top, so it would be easier to identify what about diagnostics file is?

Reply:
fixed